src/styles/main.scss

```diff
@@ -0,0 +1,77 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+$calendar-day-width: 100px;
+
+.calendar {
+  width: $calendar-day-width * 7 + 6px;
```

Comment:
The calculation for the calendar width (`$calendar-day-width * 7 + 6px`) assumes there are 6 gaps of 1px between the days. If the gap size changes or if there are more gaps, this calculation might not be accurate. Consider using `calc()` or adjusting the formula to dynamically account for the number of gaps and their size.